### DISCUSSION TITLE
Suppress warning "Cannot find" from c_cpp_properties.json

### DISCUSSION THREAD

**Discussion by FStefanni** [microsoft/vscode-cpptools#8416] [Q&A]
Hi,

I am using a `c_cpp_properties.json` file, which contains some `includePath` which do not exist (I am developing by using different versions of the same lib). So the fact that these paths do not exist is *on purpose*.
But the "Problems" tab is reporting the warning "Cannot find \<path\>", for the paths that do not exist.

Is there a way to hide this warning?

Thank you,
regards.

**Comment by sean-mcmanus** [comment]
![image](https://user-images.githubusercontent.com/19859882/142489705-2876deeb-588f-4359-a59e-74bb665fe23a.png)
or
![image](https://user-images.githubusercontent.com/19859882/142489767-1683b2c8-3931-4553-aecf-19f58d3e75b1.png)

Other than that, there isn't a way to disable a specific warning code (yet).
